DIGNDB.App.SmitteStop.API/appsettings.json

```diff
@@ -73,6 +73,59 @@
     "TransmissionRiskScores": [ 1, 1, 8, 8, 8, 8, 1, 1 ],
     "TransmissionRiskWeight": 50
   },
+  "DailySummaryConfiguration": {
+    "AttenuationThresholds": [ 50, 70, 90 ],
+    "AttenuationWeights": {
```

Comment:
Use the following weights:
"Immediate": 2,
"Near": 1,
"Medium": 0.25,
"Other": 0